internal/api_admin/server.go

```diff
@@ -0,0 +1,86 @@
+package api_admin
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/iden3/iden3comm"
+
+	"github.com/polygonid/sh-id-platform/internal/config"
+	"github.com/polygonid/sh-id-platform/internal/core/ports"
+	"github.com/polygonid/sh-id-platform/internal/health"
+)
+
+// Server implements StrictServerInterface and holds the implementation of all API controllers
+// This is the glue to the API autogenerated code
+type Server struct {
+	cfg              *config.Configuration
+	identityService  ports.IdentityService
+	claimService     ports.ClaimsService
+	schemaService    ports.SchemaService
+	publisherGateway ports.Publisher
+	packageManager   *iden3comm.PackageManager
+	health           *health.Status
```

Comment:
Not sure if we are gonna need all of these.  But it is ok.

Reply:
Well, me neither too. I left those parameters for now until we will have more information regarding what we are going to need.